run-tests.sh

```diff
@@ -13,36 +13,67 @@ set -o errexit
 # Quit on unbound symbols
 set -o nounset
 
+check_script () {
+    shellcheck run-tests.sh
+}
```

Comment:
Does the GA output looks nicer if we add those explicit echo commands? For example:

```
check_script () {
    echo "==> Checking scripts..."
    shellcheck run-tests.sh
    echo "==> Success!"
}
```

Maybe we don't need that... I thinking out loud how we could improve the output that currently in case of success looks somewhat cryptic:

![2020-10-13-143025_2560x1440_scrot](https://user-images.githubusercontent.com/517546/95860831-e4b4ae80-0d60-11eb-9ade-71d8ef771ebc.png)

Reply:
Added a black error.

Reply:
Looks good; please remove the black error and we are good to go I think

Reply:
Done 👍